### PR TITLE
SD-1562 Add close button to edit package form

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: client/spec/helpers/utils.js [hunk 1]
```diff
@@ -1,15 +1,49 @@
...
 // construct url from uri and base url
 exports.constructUrl = function(base, uri) {
     return base.replace(/\/$/, '') + uri;
 };
 
-var LoginModal = require('./pages').login;
+var currBrowser = browser;
+
+function BrowserManager() {
+    this.setBrowser = function(userBrowser) {
```
Comment (amagdas) [inline client/spec/helpers/utils.js]: Is this ever used?

Comment (ioanpocol) [inline client/spec/helpers/utils.js]: It should be used on tests. A multi-browser test should be added.

Comment (petrjasek) [inline client/spec/helpers/utils.js]: yep with test it would be better

**Comment by mugurrus** [comment]
it's just wip, sorry didn't flag it as such from the beginning

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**